xml/System.Globalization/CalendarWeekRule.xml

```diff
@@ -83,15 +83,15 @@
 |Jan 8 Fri|Week 2|Week 1|Week 1|  
 |Jan 9 Sat|Week 2|Week 1|Week 1|  
   
- The following example illustrates how the <xref:System.Globalization.CalendarWeekRule.FirstDay> and <xref:System.DayOfWeek?displayProperty=nameWithType> values are used together to determine how weeks are assigned. In the Gregorian calendar, the first day of the year (January 1) in 2013 falls on a Tuesday. If the designated first day of the week is Sunday, the first Sunday (January 6) is the first day of the first week of the year, and Saturday (January 5) belongs to the fifty-third week of the previous year. Changing the calendar week rule to FirstFourDayWeek defines Tuesday (January 1) as the beginning of the first week of the year, because there are more than four days between Tuesday, January 1, and Sunday, January 6. Using this rule, January 5 belongs to the first week of the year. For 2010, a year in which January 1 falls on a Friday, applying the FirstFourDayWeek rule with <xref:System.DayOfWeek?displayProperty=nameWithType> as the first day of the week makes Sunday, January 3 the beginning of the first week of the year, because the first week in 2010 that has more than four days is January 3 through 9.  
+ The following example illustrates how the <xref:System.Globalization.CalendarWeekRule> and <xref:System.DayOfWeek?displayProperty=nameWithType> values are used together to determine how weeks are assigned. In the Gregorian calendar, the first day of the year (January 1) in 2013 falls on a Tuesday. If the designated first day of the week is Sunday, the first Sunday (January 6) is the first day of the first week of the year, and Saturday (January 5) belongs to the fifty-third week of the previous year. Changing the calendar week rule to `FirstFourDayWeek` defines Tuesday (January 1) as the beginning of the first week of the year, because there are more than four days between Tuesday, January 1, and Sunday, January 6. Using this rule, January 5 belongs to the first week of the year. For 2010, a year in which January 1 falls on a Friday, applying the `FirstFourDayWeek` rule with <xref:System.DayOfWeek.Sunday?displayProperty=nameWithType> as the first day of the week makes Sunday, January 3 the beginning of the first week of the year, because the first week in 2010 that has more than four days is January 3 through 9.  
```

Comment:
```suggestion
 The following example illustrates how the <xref:System.Globalization.CalendarWeekRule> and <xref:System.DayOfWeek> values are used together to determine how weeks are assigned. In the Gregorian calendar, the first day of the year (January 1) in 2013 falls on a Tuesday. If the designated first day of the week is Sunday, the first Sunday (January 6) is the first day of the first week of the year, and Saturday (January 5) belongs to the fifty-third week of the previous year. Changing the calendar week rule to `FirstFourDayWeek` defines Tuesday (January 1) as the beginning of the first week of the year, because there are more than four days between Tuesday, January 1, and Sunday, January 6. Using this rule, January 5 belongs to the first week of the year. For 2010, a year in which January 1 falls on a Friday, applying the `FirstFourDayWeek` rule with <xref:System.DayOfWeek.Sunday?displayProperty=nameWithType> as the first day of the week makes Sunday, January 3 the beginning of the first week of the year, because the first week in 2010 that has more than four days is January 3 through 9.  
```